### PR TITLE
test(oomcli): add stream feature export integration test

### PR DESCRIPTION
**Pull request by lianxmfor**

### CODE REVIEW

**Review by jinghancc — APPROVED** [review]
LGTM

File: oomcli/test/test_stream_export.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+'
+actual=$(oomcli export --feature user-click.last_5_click_posts,user-click.number_of_user_starred_posts --unix-milli $t2 -o csv)
+assert_eq "$case" "$(sort <<< "$expected")" "$(sort <<< "$actual")"
+
+case="push new entity key feature "
+expected='user,user-click.last_5_click_posts,user-click.number_of_user_starred_posts
+1,"1,2,3,4,5",10
+1,"2,3,4,5,6",11
+2,"1,2,3,4,5",10
+
```
Comment (wfxr) [inline oomcli/test/test_stream_export.sh:38]: Extra empty line?

Comment (lianxmfor) [reply oomcli/test/test_stream_export.sh]: fixed

**Review by wfxr — APPROVED** [review]
lgtm

**Pull request merged**